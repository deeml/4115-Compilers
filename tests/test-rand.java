import java.util.*;

class program 
{
	//input

	//output

	private static Random __rand = new Random(System.currentTimeMillis());
	public static void main(String[] args){
		__rand = new Random(1234);
System.out.println(__rand.nextFloat());

		return ;
	}
}
