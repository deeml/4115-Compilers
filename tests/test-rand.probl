input{}
output{}
model
{
	srand(1234);
    print(rand());
}
