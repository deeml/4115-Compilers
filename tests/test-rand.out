0.6465821
